File: sdkwork-app-sdk-csharp/Models/VoiceItem.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class VoiceItem
    {
        public string? VoiceId { get; set; }
        public string? Name { get; set; }
        public string? Description { get; set; }
        public string? Language { get; set; }
        public string? Gender { get; set; }
        public string? Style { get; set; }
        public string? PreviewAudioUrl { get; set; }
    }
}

File: sdkwork-app-sdk-go/types/voice_item.go
package types

// 语音项
type VoiceItem struct {
	VoiceId string `json:"voiceId"`
	Name string `json:"name"`
	Description string `json:"description"`
	Language string `json:"language"`
	Gender string `json:"gender"`
	Style string `json:"style"`
	PreviewAudioUrl string `json:"previewAudioUrl"`
}
